sdk/core/core-rest-pipeline/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.0.2 (Unreleased)
 
+- Added a `challenge` optional property to the `bearerTokenAuthenticationPolicy` that gives it support to process [Continuous Access Evaluation](https://docs.microsoft.com/azure/active-directory/conditional-access/concept-continuous-access-evaluation) challenges.
```

Comment:
I renamed it to `challengeCallbacks` because it was causing linting errors (local variables `challenge` shadowing it) and it doesn't represent challenge?  Any better suggestions?

Reply:
yeah definitely!